### PR TITLE
affichage des questions

### PR DESCRIPTION
**Pull request by nassimcr**

### CODE REVIEW

**Comment by nassimcr** [comment]
d

File: backend/.gitignore [hunk 1]
```diff
@@ -1,2 +1 @@
 node_modules
-.env
```
Comment (rocambille) [inline backend/.gitignore:2]: .env files should not be added to github : they may contain passwords. you should keep them ignored

File: backend/.env [hunk 2]
```diff
@@ -0,0 +1,7 @@
+APP_PORT=5001
+FRONTEND_URL=http://localhost:5173
+DB_HOST=localhost
+DB_PORT=3306
+DB_USER=notroot
+DB_PASSWORD=helloworld
+DB_NAME=mvc_express
```
Comment (rocambille) [inline backend/.env]: remove this file from the repo: it should not be versioned

File: package.json [hunk 3]
```diff
@@ -28,6 +28,10 @@
   "devDependencies": {
     "concurrently": "^7.6.0",
     "cross-env": "^7.0.3",
-    "husky": "^8.0.3"
+    "husky": "^8.0.3",
+    "nodemon": "^2.0.22"
+  },
+  "dependencies": {
+    "express": "^4.18.2"
```
Comment (rocambille) [inline package.json]: this file should not be modified : restore it, and restore package-lock.json

**Review by rocambille — CHANGES_REQUESTED** [review]

**Pull request merged**